simplq/src/components/pages/Admin/QueueSettings.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import SettingsIcon from '@material-ui/icons/Settings';
+import Modal from 'components/common/Modal';
+import HighlightOffIcon from '@material-ui/icons/HighlightOff';
+import SidePanelItem from 'components/common/SidePanel/SidePanelItem';
+import { useUpdateQueueSettings } from 'store/asyncActions';
+import { useDispatch, useSelector } from 'react-redux';
+import Grid from '@material-ui/core/Grid';
+import SaveIcon from '@material-ui/icons/Save';
+import { selectMaxQueueCapacity } from 'store/selectedQueue';
+import Button from '../../common/Button';
+import InputField from '../../common/InputField';
+import styles from './QueueSettings.module.scss';
```

Comment:
Instead of long relative imports, you can use 

`import InputField from 'components/common/InputField';`

We've defined the base path to use in jsconfig.json